spec/features/consumer/shopping/unit_price_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+feature "As a consumer, I want to check unit price information for a product ;", js: true do
```

Comment:
I've never seen a description ending in ` ;`. Did you see that somewhere else?

Reply:
No. It's just a fantasy from me. Removing it.